GIT_AND_GITHUB.md

```diff
@@ -89,6 +77,13 @@ discuss other alternatives. Of course, this means you are putting yourself at ri
 of sharing your knowledge and/or learning some new stuff, so please be careful not
 to end up being even more awesome than you are.
```

Comment:
maybe make ik 'than you already are'. To point more to the process of growing an learning and becoming more awesome all the time :v: